### PR TITLE
feat(natives/gamebuild): update natives from the 2189 gamebuild

### PR DESCRIPTION
**Pull request by ahcenezdh**
Update natives from the 2189 gamebuild with names and descriptions, should be good for review @4mmonium

### CODE REVIEW

File: OBJECT/SetPropTintIndex.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+ns: OBJECT
+aliases: ["0x31574B1B41268673"]
+---
+## SET_PROP_TINT_INDEX
+
+```c
+// 0x31574B1B41268673
+void SET_PROP_TINT_INDEX(cs_type(Any) Object object, cs_type(Any) int tint);
```
Comment (4mmonium) [inline OBJECT/SetPropTintIndex.md:9]: `Vehicle`, `Object` and other related types [here](https://github.com/citizenfx/fivem/blob/030778a167242f79e0f59cd8d9c04b970e3b75c2/ext/natives/codegen_types.lua#L16-L37) map to `int` so using `cs_type` is generally not required for those. Also not required for `Any` to `int`.

File: VEHICLE/SetIgnorePlanesSmallPitchChange.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+ns: VEHICLE
+aliases: ["0x8664170EF165C4A6"]
+---
+## SET_IGNORE_PLANES_SMALL_PITCH_CHANGE
+
+```c
+// 0x8664170EF165C4A6
+void SET_IGNORE_PLANES_SMALL_PITCH_CHANGE(cs_type(Any) Vehicle vehicle, cs_type(Any) BOOL ignorePitching);
```
Comment (4mmonium) [inline VEHICLE/SetIgnorePlanesSmallPitchChange.md:9]: Same for this, notice [this](https://github.com/citizenfx/fivem/blob/030778a167242f79e0f59cd8d9c04b970e3b75c2/ext/natives/codegen_types.lua#L16-L37).

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey so far this looks almost OK, left you some comments regarding `cs_type` usage. Thanks!

**Comment by ahcenezdh** [comment]
Hey @4mmonium happy holidays to you 🎉and should be good, if you could check my issue #961 would be awesome!

File: PATHFIND/SetAllowStreamHeistIslandNodes.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+ns: PATHFIND
+aliases: ["_SET_AI_GLOBAL_PATH_NODES_TYPE"]
+---
+## SET_ALLOW_STREAM_HEIST_ISLAND_NODES
+
+```c
+// 0xF74B1FFA4A15FBEA
+void SET_ALLOW_STREAM_HEIST_ISLAND_NODES(int type);
```
Comment (4mmonium) [inline PATHFIND/SetAllowStreamHeistIslandNodes.md:9]: That parameter is actually of `bool` type, you may name it `bAllow` I suppose.

Would change to the following:
`void SET_ALLOW_STREAM_HEIST_ISLAND_NODES(cs_type(int) BOOL bAllow)`

File: PATHFIND/SetAllowStreamHeistIslandNodes.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+Enables or disables the streaming of road nodes for the Heist Island, specifically Cayo Perico. Setting type to 1 activates the path nodes, allowing GPS navigation to function on the island. However, setting it to 0 to deactivate these nodes, especially when near the island, may cause the game to crash. Note that setting this to true also disables main level and prologue nodes from streaming, so it should be set back to false when no longer needed.
+
+```
+NativeDB Introduced: v2189
+```
+
+## Parameters
+* **type**: Pass `1` to enable Heist Island road nodes, `0` to disable them.
```
Comment (4mmonium) [inline PATHFIND/SetAllowStreamHeistIslandNodes.md:19]: Change to `true`/`false` respectively, since boolean.

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Couple more changes needed, guess I may have overlooked that, thanks!

**Comment by ahcenezdh** [comment]
Should be good 👍

**Comment by ahcenezdh** [comment]
I just removed useless return value

**Comment by ahcenezdh** [comment]
Description of `SET_IGNORE_PLANES_SMALL_PITCH_CHANGE` and `SET_ALLOW_STREAM_HEIST_ISLAND_NODES` have been revised to clarify their purpose

**Pull request closed without merging**